stdlib/2/md5.pyi

```diff
@@ -1,9 +1,16 @@
 # Stubs for Python 2.7 md5 stdlib module
 
+from typing import AnyStr
+
 class md5(object):
-    def update(self, arg: str) -> None: ...
-    def digest(self) -> str: ...
-    def hexdigest(self) -> str: ...
+    name = ...  # type: str
+    block_size = ...  # type: int
+    digest_size = ...  # type: int
+    digestsize = ...  # type: int
+    def __init__(self, arg: AnyStr= ...) -> None: ...
+    def update(self, arg: AnyStr) -> None: ...
+    def digest(self) -> AnyStr: ...
```

Comment:
It doesn't make sense to use `AnyStr` only in the return type of a method -- a type checker won't be able to infer a value for the type variable, as there's no context available. Here the correct return type seems to be just `str`, as I can't get this to return a `unicode` object.